reference/customers.v3.yml

```diff
@@ -528,7 +528,34 @@ paths:
       tags:
         - Customer Addresses
     post:
-      description: Creates a Customer Address. Multiple customer addresses can be created in one call.
+      description: |-
+        Creates a Customer Address. Multiple customer addresses can be created in one call.
+
+        **Required Fields**
+        * **customer_id**
+        * **first_name**
+        * **last_name**
+        * **city**
+        * **country_code**
+        * **state_or_province**
+        * **address1**
+        * **postal_code**
```

Comment:
Wait, is postal code required? We have logic in place to make it optional based on if country actually requires the post code. Same actually is true for the state.